models/users.js

```diff
@@ -8,6 +8,24 @@ Users.initEasySearch(searchInFields, {
   returnFields: [...searchInFields, 'profile.avatarUrl'],
 });
 
+if (Meteor.isClient) {
```

Comment:
These helpers access `Session` which is only defined on the client. Therefor they should be defined on the client only. Using them on the server would throw an error because `Session` is not defined there.